src/internal/connector/exchange/api/mail.go

```diff
@@ -145,8 +150,16 @@ func (c Mail) GetItem(
 		return nil, nil, graph.Stack(ctx, err)
 	}
 
-	if !ptr.Val(mail.GetHasAttachments()) && !HasAttachments(mail.GetBody()) {
-		return mail, MailInfo(mail), nil
+	mailBody = mail.GetBody()
+	if !ptr.Val(mail.GetHasAttachments()) && !HasAttachments(mailBody) {
+		return mail, MailInfo(mail, 0), nil
```

Comment:
What does `mailBody.Size()` return? If that has the correct size including the attachments(don't think so, as no size was returning when getting emails), we can just use this within `MailInfo` instead of having to get strlen and attachment size.

Reply:
mailBody.Size() isn't a supported thing We have to either take size of content. Because attachments could not be checked here.

Reply:
as of now I have used `size of content` as the size here. Which seems logical because we will be able to backup only content at this point.